inventory/host_vars/local_vagrant/config.yml

```diff
@@ -11,7 +11,6 @@ db_password: 'vagrant123'
 admin_password: 'spree123'
 secret_token: '511a3d0fa1551b9fa75a1aef5b47684905c64807963fa2c190272878366365'
 google_maps_api_key: 'xxx'
-newrelic_license_key: 'xxx'
```

Comment:
why this?

Reply:
Because we merged https://github.com/openfoodfoundation/ofn-install/pull/211 and it's not used anymore.

Reply:
I should have separated that into a different commit to explain it.

Reply:
Oops, I read something else. I didn't notice it was newrelice 🙈